README.md

```diff
@@ -51,8 +51,8 @@ you have `OAUTH_CLIENT_ID` and `OAUTH_CLIENT_SECRET` set, you can read in a CIS
 
 ```python
 >>> from cis_publishers.common import Profile
->>> april = Profile(email="apking@mozilla.com")
->>> april
+>>> jdoe = Profile(email="jdoe@mozilla.com")
+>>> jdoe
 {'access_information': {'access_provider': None, 'hris': {'egencia_pos_country': 'US', 'employee_id': '123456', 'managers_primary_work_email': '...'}
```

Comment:
I feel compelled to mention that since we dropped Egencia, the field `egencia_pos_country` might start to deviate from reality over time. It's also possible they'll choose to maintain it, but duly noted.